sw/device/tests/entropy_src_fw_override_test.c

```diff
@@ -377,10 +442,23 @@ bool test_main(void) {
 
   for (size_t i = 0; i < ARRAYSIZE(kModes); i++) {
     EXECUTE_TEST(test_result, firmware_override_extract_insert, kModes[i],
+                 false, false);
+    EXECUTE_TEST(test_result, firmware_override_extract_insert, kModes[i], true,
                  false);
-    EXECUTE_TEST(test_result, firmware_override_extract_insert, kModes[i],
-                 true);
   }
+  // Rerun the test with single bit mode disabled, this time we reenable the
```

Comment:
Thanks for adding these two test cases on top of the existing test cases. I think it's good to keep testing the previous cases in their original form.